Integrations/integration-Quinc.yml

```diff
@@ -0,0 +1,341 @@
+configuration:
+- display: ""
+  name: ServerName
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: ""
+  name: token
+  defaultvalue: ""
+  type: 0
```

Comment:
`token` type must be encrypted and not plain short text

Reply:
In our case it is GUID, so it doesn't have to be encrypted, I suppose